solr/core/src/java/org/apache/solr/uninverting/FieldCacheImpl.java

```diff
@@ -1076,10 +1076,6 @@ public BinaryDocValues getTerms(LeafReader reader, String field) throws IOExcept
 
   public BinaryDocValues getTerms(LeafReader reader, String field, float acceptableOverheadRatio) throws IOException {
     BinaryDocValues valuesIn = reader.getBinaryDocValues(field);
-    if (valuesIn == null) {
```

Comment:
I'm removing this part. It seems to me the SortedDocValues are returned by getTermsIndex() so we don't need this block.